deps/build.jl

```diff
@@ -1,78 +0,0 @@
-const PROJECT_DIR = (@__DIR__) |> dirname
```

Comment:
Maybe keep this build script as a backup for failing to download binaries from `JuliaBinaryWrappers/LibTorchCAPI_jll.jl`? It's also useful for macOS at the moment

Reply:
Update, it can be built on macOS now.